app/src/main/res/layout/fragment_profile.xml

```diff
@@ -124,13 +127,13 @@
                     <TextView
                         android:layout_width="125dp"
```

Comment:
don't hardcode widths like this. can go wrong on smaller screens. use wrap_content